### PR TITLE
Omaster

### PR DESCRIPTION
**Pull request by MareoRaft**
These are the changes I was referencing in Issue #1 .  Let me know what you think about this.

I typically have a separate dev and prod dockerfile, but we might not need it, if you prefer the simplicity of having just one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
 sudo docker build -t analythium/shinyproxy-demo .
 ```
 
-Test locally
+
+
+## Develop and test locally
+```bash
+docker-compose up --build
```
Comment (MareoRaft) [inline README.md:28]: `docker-compose up --build` as opposed to the previous `docker run -p 4000:3838 analythium/shinyproxy-demo`.

one of the main differences is that now when you make changes to your app, reloading the page is enough to immediately see the changes.

File: README.md [hunk 2]
```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
...
+To build the image from the Dockerfile, run
 ```bash
-docker run -p 4000:3838 analythium/shinyproxy-demo
+sudo docker build -f prod.Dockerfile -t yourregistryusername/shinyproxy-test:prod .
 ```
-then visit `127.0.0.1:4000`.
+
+then run locally with
+```bash
+docker run -p 4000:3838 yourregistryusername/shinyproxy-test:prod
```
Comment (MareoRaft) [inline README.md:45]: this should say `shinyproxy-demo`.  feel free to fix these if you approve everything else.  sorry about that.

File: README.md [hunk 3]
```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
...
+## Develop and test locally
+```bash
+docker-compose up --build
+```
+then visit `http://localhost:4000`.  To develop, simply make changes to your Shiny app and then reload the page!
+
+
+
+
+## Production build, test, and deploy
```
Comment (MareoRaft) [inline README.md:35]: we don't have to include deployment docs at all if you don't want.  i just found it helpful for myself.

File: README.md [hunk 4]
```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
...
+then run locally with
+```bash
+docker run -p 4000:3838 yourregistryusername/shinyproxy-test:prod
+```
+and visit `http://localhost:4000` to double check it works.  Finally, deploy with
+
+```bash
+docker push yourregistryusername/shinyproxy-test:prod
+cd ~/programming/shinyproxy-1-click/digitalocean
+bash setup.sh -i ~/.ssh/id_rsa -s root@yourserveripaddress -f /path/to/application.yml
```
Comment (MareoRaft) [inline README.md:52]: the setup.sh script was in your other repo, not this one

**Comment by psolymos** [comment]
Closes #1

**Pull request merged**